softdevice_controller/CHANGELOG.rst

```diff
@@ -53,10 +67,14 @@ Changes
   Using :file:`mpsl_ecb.h` is now recommended instead. (DRGN-21603)
 * The ability to configure a periodic advertiser with subevents but without response slots has been removed.
   This is due to an errata to the Bluetooth Core Specification v5.4 no longer allowing this configuration. (DRGN-22189)
+* The vendor-specific Set Connection Event Trigger command can now be used with :kconfig:option:`CONFIG_BT_LL_SOFTDEVICE_PERIPHERAL`. (DRGN-22511)
```

Comment:
I don't think this and l 75, 76 and 77 made it into 2.7, may need to be moved to Main branch.

Reply:
True, will remove